### ISSUE TITLE
Add searching and suggesting trending hashtags during sign-up

### ISSUE THREAD

**Issue by Freeplayg** [mastodon/mastodon#20451]
### Pitch

Add a page during the sign-up process, to be able to search through popular hashtags that you can then follow

### Motivation

I think it's pretty hard for new users to find content they like, so I think letting people search through and follow hashtags that may interest them may be a good idea (showing most active on top) to make it easier for people to find content they're interested in.

**Comment by outfrost** [comment]
I like this, though I feel "popular" is a better word here than "trending". A news story that briefly spikes the use of a hashtag maybe isn't something I'm _usually_ interested in (though I will search for it if I am), but a video game that got popular a month ago and still has people talking about it might be a conversation topic I'm looking for.

Also, everyone's interests are different, so a bit of randomisation (especially beyond the first few "most popular" hashtags) might be a good idea.

**Comment by SnappyTwig** [comment]
This can also ease the biggest challenge to new users: instance selection. 

You could suggest instances to users by frequency of users chosen hashtags in that instance’s feed:

https://github.com/mastodon/mastodon/issues/20854#issuecomment-1320884438

**Comment by supernovae** [comment]
I agree the "empty timeline" problem is a big problem and that guiding users to follow 3-5 hashtags would be an awesome feature whether we recommend what is trending or ask them 1-3 topics to follow and turn those into #hashtags

**Comment by brendanjones** [comment]
https://github.com/mastodon/mastodon/issues/20854 is duplicate I think but there's more discussion over there now, so maybe close this one even though it predates that one?
